src/test/java/org/reactome/release/common/dataretrieval/COSMICFileRetrieverIT.java

```diff
@@ -0,0 +1,51 @@
+package org.reactome.release.common.dataretrieval;
+
+import static org.junit.Assert.fail;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.Duration;
+import java.util.Properties;
+
+import org.junit.Test;
+import org.reactome.release.common.dataretrieval.cosmic.COSMICFileRetriever;
+
+public class COSMICFileRetrieverIT
+{
+
+	
+	@Test
+	public void testGetCOSMICData() throws URISyntaxException, FileNotFoundException, IOException
+	{
+		// to run this test, you will need to create your own auth.properties under src/test/resources
+		// and populate it with a valid username and password for COSMIC. 
+		Properties props = new Properties();
+		props.load(new FileReader("src/test/resources/auth.properties"));
+		COSMICFileRetriever retriever = new COSMICFileRetriever();
+		retriever.setUserName(props.getProperty("username"));
+		retriever.setPassword(props.getProperty("password"));
+		
+		// classification.csv is a pretty small file, ~2MB I think, so a good choice for testing downloads.
+		retriever.setDataURL(new URI("https://cancer.sanger.ac.uk/cosmic/file_download/GRCh38/cosmic/v92/classification.csv"));
```

Comment:
Just commenting to point out the v92 in the string - I know there isn't much that can be done about it as far as I know, but would be good to at least try in the future to have it worked with the current release, or the release that we download the cosmic files from. Could maybe be put as a properties? (not this PR, just thinking)

Reply:
Maybe, though this is testing code so it really just cares that some download works.

Reply:
Fair enough, wasn't sure if we would at some point want it to focus on the current release, but it definitely doesn't matter at this point in time